sdks/python/apache_beam/tests/pipeline_verifiers.py

```diff
@@ -48,3 +55,63 @@ def describe_mismatch(self, pipeline_result, mismatch_description):
     mismatch_description \
       .append_text("Test pipeline job terminated in state: ") \
       .append_text(pipeline_result.current_state())
+
+
+def retry_on_fileio_error(exception):
+  """Filter allowing retries on file I/O errors."""
+  if isinstance(exception, RuntimeError) or \
+          isinstance(exception, IOError):
+    # GCS I/O raises RuntimeError and local filesystem I/O
```

Comment:
Create [JIRA](https://issues.apache.org/jira/browse/BEAM-1264). Will remove all reference to GCS